app/cli.php

```diff
@@ -219,6 +219,13 @@
 }, ['publisher']);
 CLI::setResource('logError', function (Registry $register) {
     return function (Throwable $error, string $namespace, string $action) use ($register) {
+        Console::error('[Error] Timestamp: ' . date('c', time()));
+        Console::error('[Error] Type: ' . get_class($error));
+        Console::error('[Error] Message: ' . $error->getMessage());
+        Console::error('[Error] File: ' . $error->getFile());
+        Console::error('[Error] Line: ' . $error->getLine());
+        Console::error('Trace: ' . $error->getTraceAsString());
```

Comment:
Let's add the error tag here too:

```suggestion
        Console::error('[Error] Trace: ' . $error->getTraceAsString());
```